README.md

```diff
@@ -24,9 +24,18 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for ways to get started.
 - [dnd kit](https://dndkit.com/)
 - [react-markdown](https://github.com/remarkjs/react-markdown)
 
+## Installing/Updating NextJS:
+First step is being sure your running the latest [NEXTJS](https://github.com/vercel/next.js):
```

Comment:
Just a couple wording/capitalization improvements:
```suggestion
The first step is to make sure you're running the latest [Next.js](https://github.com/vercel/next.js):
```

Reply:
Great suggestions, thank you